src/bentoml/_internal/utils/unflatten.py

```diff
@@ -39,48 +39,47 @@
 from __future__ import absolute_import
 
 import re
-import sys
 import typing as t
 from operator import itemgetter
 
-if sys.version_info[0] == 2:
-    string_type = basestring  # noqa: F821
-else:
-    string_type = str
-
 
 def unflatten(arg: dict[str, t.Any]) -> dict[str, t.Any]:
     """Unflatten nested dict/array data.
 
     This function takes a single argument which may either be a
```

Comment:
Let's leave styles here along, so that it can be sync with upstream easier in the future.

Reply:
Should we have a vendor folder that specifically adds all third-party usage there?

Reply:
@aarnphm I think it's nice to have?

Reply:
I can make a PR for that. then we can separate it out the dependencies from vendor vs internal utils.